gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FlowSpec.java

```diff
@@ -146,14 +150,13 @@ public static FlowSpec.Builder builder(URI catalogURI, Properties flowProps) {
    * @param key
    * @param value
    */
-  public synchronized void addProperty(String key, String value) {
+  public void addProperty(String key, Long value) {
```

Comment:
why do you remove `synchronized`... don't these two updates need to happen in lockstep?

Reply:
missed that during refactoring, added back